README.md

```diff
@@ -308,13 +321,37 @@ Add the version of `solc` you want to use into `index.html`:
 ```html
 <script
   type="text/javascript"
-  src="https://binaries.soliditylang.org/bin/{{ SOLC VERSION }}.js"
+  src="https://binaries.soliditylang.org/bin/{{ SOLC_VERSION }}.js"
+  integrity="sha256-{{ BASE64_ENCODED_HASH_OF_SOLC_VERSION  }}"
+  crossorigin="anonymous"
 ></script>
 ```
 
-(Alternatively use `https://binaries.soliditylang.org/bin/soljson-latest.js` to get the latests version.)
+This will load `solc` into the global variable `window.Module`.
+Alternatively, use `soljson-latest.js` as `{{ SOLC_VERSION }}.js` in the code snippet above to load the latest version.
+
+It is recommended that you check the integrity of the resource being fetched before using it in your application.
+For that, you can use the [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) feature.
+Adding SRI configuration to your HTML script tag ensures that the resource will only be loaded by the browser if the cryptographic hashes match.
+
+You can generate the SRI hash yourself based on the base64-encoded version of the sha256 hash of the release.
+For example, after downloading version `v0.8.16+commit.07a7930e`, run:
+```bash
+shasum -b -a 256 soljson-v0.8.16+commit.07a7930e.js | awk '{ print $1 }' | xxd -r -p | base64
```

Comment:
I think `xxd` may be a bit obscure (on my system it comes with gvim :)). Personally, I do use it, but for our users, who are likely JS developers perhaps it would be better to suggest a JS one-liner via `node -e` instead?

Reply:
> on my system it comes with gvim
Really? Weird! haha

Ok. I think that the use of `xxd` is fine for the sake of an example. But I updated it to use node as you suggested.